include/z64save.h

```diff
@@ -438,7 +438,7 @@ typedef enum {
 #define EVENTCHKINF_4C 0x4C
 #define EVENTCHKINF_4D 0x4D
 #define EVENTCHKINF_4E 0x4E
-#define EVENTCHKINF_4F 0x4F
+#define EVENTCHKINF_WATCHED_SHEIK_AFTER_MASTER_SWORD_CS 0x4F
```

Comment:
Could you add a comment here to clarify which cutscene
Maybe

```suggestion
#define EVENTCHKINF_WATCHED_SHEIK_AFTER_MASTER_SWORD_CS 0x4F // Cutscene in Temple of Time as adult after the first time pulling the Master Sword
```

Reply:
I can, but im not sure what this can be confused with. there is only one cutscene with sheik after the master sword

Reply:
Yeah if you know all the cutscenes in the game "sheik after master sword" is enough, but if you don't more context doesn't hurt